demos/python_demos/language_model_demo/lm_1b_sample.py

```diff
@@ -0,0 +1,222 @@
+#!/usr/bin/env python
```

Comment:
Please rename `lm_1b_sample.py` to `language_model_demo.py`
Also please read [contribution guide](https://github.com/opencv/open_model_zoo/blob/develop/CONTRIBUTING.md#demo)